msteams-platform/tabs/how-to/authentication/tab-sso-graph-api.md

```diff
@@ -213,25 +213,30 @@ If you need to access Microsoft Graph data, configure your server-side code to:
 > 
 > * Don’t use `notifySuccess` result to return the token information to the parent page. Use `localStorage` to save the token and pass the item key via `notifySuccess`.
 
-## Known limitations
+## Obtaining consent
 
 Tenant admin consent: A simple way of [consenting on behalf of an organization as a tenant admin](/azure/active-directory/manage-apps/consent-and-permissions-overview#admin-consent) is by getting [consent from admin](/azure/active-directory/manage-apps/grant-admin-consent).
 
-You can ask for consent using the Auth API. Another approach for getting Graph scopes is to present a consent dialog using our existing [third party OAuth provider authentication approach](~/tabs/how-to/authentication/auth-tab-aad.md#navigate-to-the-authorization-page-from-your-pop-up-page). This approach involves popping up an Azure AD consent dialog box.
+Another approach for getting Graph scopes is to present a consent dialog using our existing [third party OAuth provider authentication approach](~/tabs/how-to/authentication/auth-tab-aad.md#navigate-to-the-authorization-page-from-your-pop-up-page). This approach involves popping up an Azure AD consent dialog box.
```

Comment:
I'm actually a little confused on what this "use our existing third party...approach" means here? Is this saying "If you don't want to use OBO, then you can just directly show UI to sign the user into AAD and get a token that lets you directly access the graph?"

If that is true, is there any guidance we could give developers about why they would prefer that approach over OBO? Do we have a preference on which is a better experience (e.g., better user experience, covers more auth cases, etc.)?

Reply:
Hmm, good questions. @iambmelt would you have any suggestions?

Reply:
I don't understand the intent behind this (existing) statement either, and I don't think it makes sense in the current article. Its purpose seems to introduce the steps of asking the user for additional consent. I'll take a stab at reworking this--